docs/evaluate/temporal-cloud/pricing.mdx

```diff
@@ -29,182 +29,408 @@ tags:
 ---
 
 Temporal Cloud is a consumption-based service.
-You pay only for what you need, when you need it.
-Our pricing reflects your use of _actions_, _storage_, and _support_.
-It is flexible, transparent, and predictable so you know your costs and never pay for unused capacity.
+You pay only for what you use.
+Our pricing reflects your use of _Actions_, _Storage_, and _Support_.
+It is flexible, transparent, and predictable so you know your costs.
 
-This page introduces the elements of Temporal Cloud pricing for [actions](#action), [storage](#storage), and [services/support](#support).
-It gives you the information you need to estimate costs for your implementation.
-For more accurate estimates, [reach out to our team](https://pages.temporal.io/contact-us). Once you have an account, you can view your cost and billing details in the account directly. For more information, view [Billing and Cost](/cloud/billing-and-cost).
+This page describes the elements of Temporal Cloud pricing for [Actions](#action), [Storage](#storage), and [Support](#support).
```

Comment:
Can we combine line 33 and 36? It seems pretty repetitive

Reply:
Removing words from "for [Actions]..." forward and moving links to 33